<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ADMIN" failures="2" tests="16" name="RetryFailedTestCases" time="0.168" errors="0" timestamp="3 Aug 2022 12:53:10 GMT">
  <testcase classname="testng_Failed.FirstClassTest" name="testMethod1" time="0.019"/>
  <testcase classname="testng_Failed.FirstClassTest" name="testMethod2" time="0.002"/>
  <testcase classname="testng_Failed.FirstClassTest" name="testMethod3" time="0.005"/>
  <testcase classname="testng_Failed.FirstClassTest" name="testMethod4" time="0.003"/>
  <testcase classname="testng_Failed.FirstClassTest" name="testMethod5" time="0.006">
    <skipped/>
  </testcase> <!-- testMethod5 -->
  <testcase classname="testng_Failed.FirstClassTest" name="testMethod5" time="0.002">
    <skipped/>
  </testcase> <!-- testMethod5 -->
  <testcase classname="testng_Failed.FirstClassTest" name="testMethod5" time="0.007">
    <skipped/>
  </testcase> <!-- testMethod5 -->
  <testcase classname="testng_Failed.FirstClassTest" name="testMethod5" time="0.009">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
	at testng_Failed.FirstClassTest.testMethod5(FirstClassTest.java:37)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- testMethod5 -->
  <testcase classname="testng_Failed.SecondClassTest" name="testMethod1" time="0.007"/>
  <testcase classname="testng_Failed.SecondClassTest" name="testMethod2" time="0.005"/>
  <testcase classname="testng_Failed.SecondClassTest" name="testMethod3" time="0.003"/>
  <testcase classname="testng_Failed.SecondClassTest" name="testMethod4" time="0.002"/>
  <testcase classname="testng_Failed.SecondClassTest" name="testMethod5" time="0.002">
    <skipped/>
  </testcase> <!-- testMethod5 -->
  <testcase classname="testng_Failed.SecondClassTest" name="testMethod5" time="0.002">
    <skipped/>
  </testcase> <!-- testMethod5 -->
  <testcase classname="testng_Failed.SecondClassTest" name="testMethod5" time="0.003">
    <skipped/>
  </testcase> <!-- testMethod5 -->
  <testcase classname="testng_Failed.SecondClassTest" name="testMethod5" time="0.002">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
	at testng_Failed.SecondClassTest.testMethod5(SecondClassTest.java:35)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- testMethod5 -->
</testsuite> <!-- RetryFailedTestCases -->
